feat: Add Svelte SDK docs (#5413)
getsentry/sentry-docs: src/includes/sourcemaps/generate/javascript.svelte.mdx

> To generate source maps with your Svelte project, you need to set the sourcemap [compiler options](https://svelte.dev/docs#compile-time-svelte-compile) in your Svelte config:

Reviewer: If you're using the term "source map" generally here, then go with the edit I've made. If you're trying to reference the sourcemap option in Svelte more directly, then change it to `sourcemap` in code format.

```suggestion
To generate source maps with your Svelte project, you need to set the source map [compiler options](https://svelte.dev/docs#compile-time-svelte-compile) in your Svelte config:
```

```JavaScript {filename:svelte.config.js}
import sveltePreprocess from "svelte-preprocess";

const config = {
  compilerOptions: {
    enableSourcemap: true,
  },
  preprocess: sveltePreprocess({
    sourceMap: true,
  }),
};

export default config;
```

> Additionally, you need to enable sourcemap output in your bundler. The following example shows how to do this for Vite:

Reviewer: Same comment as above about the term "sourcemap" here.

```JavaScript {filename:vite.config.js}
import { defineConfig } from "vite";
import { svelte } from "@sveltejs/vite-plugin-svelte";

export default defineConfig({
  plugins: [svelte()],
  build: {
    sourcemap: true,
  },
});
```

> If you are using other bundlers than Vite, please refer to our general guide how to [configure source maps generation](../../../../sourcemaps/generating) with them or to their documentation.

Reviewer: ```suggestion
If you're using bundlers other than Vite, check out our general guide on how to [generate source maps](../../../../sourcemaps/generating) with them, or refer to their documentation.
```
